Makefile

```diff
@@ -1,17 +1,24 @@
 # This file is for you! Edit it to implement your own hooks (make targets) into
 # the project as automated steps to be executed on locally and in the CD pipeline.
 
-include scripts/init.mk
+include ./scripts/init.mk
 
-# ==============================================================================
-
-# Example CI/CD targets are: dependencies, build, publish, deploy, clean, etc.
+# Example targets are: dependencies, build, publish, deploy, clean, etc.
 
 dependencies: # Install dependencies needed to build and test the project @Pipeline
 	# TODO: Implement installation of your project dependencies
 
-build: # Build the project artefact @Pipeline
-	# TODO: Implement the artefact build step
+build:  # Build the project artefact @Pipeline
```

Comment:
I'm not sure we should add the project make targets at this point. I suggest we postpone this 1) until we decide how to support a specific tech stack, or 2) we create a separate PR, as I think more work is needed.

Reply:
It feels like we can afford to have these here on the basis that it follows the same structure as the test scripts.  We're allowing the tech stacks to define prebaked `scripts/projecthooks/whatever.sh`.